files/usr/lib/jeos-firstboot

```diff
@@ -194,17 +198,20 @@ if [ -z "$KEYTABLE_PRESET" ]; then
         && d --default-item "$default" --menu  $"Select Keyboard Layout" 0 0 $dh_menu "${list[@]}"; then
         if [ -n "$result" ]; then
             keytable="$result"
-
-                # Activate the selected keyboard layout
-                run langset.sh "$locale" "$keytable" || warn $"Setting the keyboard layout failed"
         fi
     else
         d --msgbox $"Error setting keyboard" 0 0
     fi
 else
-    keytable="$result"
+    keytable="$JEOS_KEYTABLE"
```

Comment:
`$keytable` can be replaced by `$JEOS_KEYTABLE` everywhere.

Reply:
Same, as the keytable can come from the menu or from the environment.

Reply:
Removed in the last PR